setup.cfg

```diff
@@ -28,7 +28,7 @@ include_package_data = true
 py_modules = inflect
 python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
 install_requires =
-    importlib_metadata
+    importlib-metadata;python_version<"3.8"
```

Comment:
The [canonical name is `importlib_metadata`](https://gitlab.com/python-devs/importlib_metadata/blob/08a3782fc382d4d2671da66a3bc0e11695d091f1/setup.cfg#L2) so it's been my preference to specify the name as indicated by the package. Probably we should figure out what is the recommended name to use here. I don't know why the names are normalized, so it's hard to understand whether the normalization is something that's meant to be replicated by consumers.